asdf/tests/test_info.py

```diff
@@ -0,0 +1,359 @@
+import getpass
+import io
+import os
+import pathlib
+import re
+import shutil
+import sys
+import tempfile
+
+import numpy as np
+import pytest
+from jsonschema.exceptions import ValidationError
+
+import asdf
+from asdf import config_context, extension, get_config, resolver, schema, treeutil, versioning
+from asdf.exceptions import AsdfDeprecationWarning, AsdfWarning
+from asdf.extension import ExtensionManager, ExtensionProxy, ManifestExtension
+from asdf.resource import DirectoryResourceMapping
```

Comment:
```suggestion
import os
import pathlib
import re
import shutil
import tempfile

import numpy as np

import asdf
from asdf.extension import ExtensionManager, ExtensionProxy, ManifestExtension
from asdf.resource import DirectoryResourceMapping
```

This should fix the other flake8 issues

Reply:
done.